prompttools/structures/config.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
```

Comment:
This config file is unnecessary. You can define relevant parameters directly in your example / test.